prototypes/index.js

```diff
@@ -319,6 +321,22 @@ const bookPrompts = {
 
     /* CODE GOES HERE */
 
+    // Annotation:
+    // Write your annotation here as a comment
+  },
+
+  getBooksByYear(books, year) {
```

Comment:
This function is super similar to the above one, but I set it up to accept a year as an argument instead. Maybe it's okay that it's similar and will help students compare and contrast functionality with arguments? 🤷

Reply:
For most of the problems, the dataset array (`books` in this case) is usually not passed in as an argument. I think the convention in jsFun tends to be `getBooksByYear(year)`. Is there a need to pass in `books` as an argument in this case?

Of course, correct me if I'm wrong on that convention too.

Reply:
I only made it like that because the other functions in this 'books' set already require the books to be passed through as an argument. There's an instruction for the previous function "removeViolence" that requests that books be accessed through a parameter and then the test treats "getNewBooks" the same way. It feels a little awkward to me to have it like this, so I am more than happy to change these so that they work without that 'books' parameter. Either way!